erizo/src/erizo/thread/Scheduler.cpp

```diff
@@ -43,6 +45,10 @@ void Scheduler::serviceQueue() {
       Function f = task_queue_.begin()->second;
       task_queue_.erase(task_queue_.begin());
 
+      if (!task_queue_.empty()) {
+        current_timeout_ = task_queue_.begin()->first;
```

Comment:
we wouldn't need to set `current_timeout_` here, am I right?

Reply:
I'm starting to think we should still set it to the current worker but also signal the change to all other workers so they can update their timeout. But perhaps I'm overthinking it?